src/test/java/com/epam/tc/hw2/ex1/HomePageTest.java

```diff
@@ -0,0 +1,83 @@
+import org.testng.annotations.Test;
+
+@Test
+public class HomePageTest {
+
+    private WebDriver driver;
+
+    @BeforeMethod
+    public void setupDriver() {
+        WebDriverManager.chromedriver().setup();
+    }
+
+    @Test
+    public void testHomePage() {
+        driver = new ChromeDriver();
+        SoftAssertions softAssertions = new SoftAssertions();
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(4, TimeUnit.SECONDS);
+        String url = "https://jdi-testing.github.io/jdi-light/index.html";
+        driver.navigate().to(url);
+        softAssertions.assertThat(driver.getCurrentUrl()).isEqualTo(url);
+        softAssertions.assertThat(driver.getTitle()).isEqualTo("Home Page");
+        driver.findElement(By.xpath("/html/body/header/div/nav/ul[2]/li/a")).click();
+        driver.findElement(By.xpath("//*[@id=\"name\"]")).sendKeys("Roman");
+        driver.findElement(By.id("password")).sendKeys("Jdi1234");
+        driver.findElement(By.id("login-button")).click();
+        WebElement webElement = driver.findElement(By.xpath("//*[@id='user-name']"));
+        softAssertions.assertThat(webElement.getText()).isEqualTo("ROMAN IOVLEV");
+        webElement = driver.findElement(By.xpath("/html/body/header/div/nav/ul[1]/li[1]/a"));
+        softAssertions.assertThat(webElement.getText()).isEqualTo("HOME");
+        webElement = driver.findElement(By.xpath("/html/body/header/div/nav/ul[1]/li[2]/a"));
+        softAssertions.assertThat(webElement.getText()).isEqualTo("CONTACT FORM");
+        webElement = driver.findElement(By.xpath("/html/body/header/div/nav/ul[1]/li[3]/a"));
+        softAssertions.assertThat(webElement.getText()).isEqualTo("SERVICE");
+        webElement = driver.findElement(By.xpath("/html/body/header/div/nav/ul[1]/li[4]/a"));
+        softAssertions.assertThat(webElement.getText()).isEqualTo("METALS & COLORS");
+        List<WebElement> imagesOnPage = driver.findElements(By.cssSelector(".benefit-icon"));
+        softAssertions.assertThat(imagesOnPage.size()).isEqualTo(4);
+        List<WebElement> textUnderImages = driver.findElements(By.cssSelector(".benefit-txt"));
+        String firstTextUnderImage = "To include good practices\nand ideas from successful\nEPAM project";
+        softAssertions.assertThat(textUnderImages.get(0).getText()).isEqualTo(firstTextUnderImage);
+        String secondTextUnderImage = "To be flexible and\ncustomizable";
+        softAssertions.assertThat(textUnderImages.get(1).getText()).isEqualTo(secondTextUnderImage);
+        String thirdTextUnderImage = "To be multiplatform";
+        softAssertions.assertThat(textUnderImages.get(2).getText()).isEqualTo(thirdTextUnderImage);
+        String fourthTextUnderImage = "Already have good base\n(about 20 internal and\nsome external projects),"
+            + "\nwish to get more…";
+        softAssertions.assertThat(textUnderImages.get(3).getText()).isEqualTo(fourthTextUnderImage);
+
```

Comment:
Please put all expected texts into one list, convert list of web elements to list of String and compare two collections

Reply:
done